src/shared/components/ErrorBoundaries/ErrorBoundaries.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * ErrorBoundaries component
+ * Catch Javascript Error
+ *
+ * @author HuyTrinh<dinhhuyams@gmail.com>
+ *
+ */
+
+import * as React from 'react'
+
+class ErrorBoundaries extends React.Component {
+	state = { error: null, errorInfo: null }
+
+	componentDidCatch(error: any, errorInfo: any) {
+		// Catch errors in any components below and re-render with error message
+		this.setState({
+			error,
+			errorInfo,
+		})
+	}
+
+	render() {
+		if (this.state.errorInfo) {
+			return (
+				<div className="error-boundaries">
+					<h2 className="error-boundaries__header">Something went wrong</h2>
+					<details style={{ whiteSpace: 'pre-wrap' }}>
+						{this.state.error && this.state.error.toString()}
+						<br />
+						{this.state.errorInfo.componentStack}
+					</details>
```

Comment:
IE & Ms.Edge do not support this tag. So let's replace it with another tag
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#Browser_compatibility

Reply:
It is not fully supported but it works ok in Edge. Edge just treats it as a <p/> tag or something